lib/syskit/roby_app/configuration.rb

```diff
@@ -81,6 +81,13 @@ class Configuration
             # @deprecated Unused, kept here for historical reasons
             attr_predicate :use_only_model_pack?, true
 
+            # Controls whether Models::TaskContext.model_for raises if there is more
+            # than one Syskit model matching the given orogen model
+            #
+            # This is false for backward compatibility reasons, and will eventually
+            # become the default behavior.
+            attr_predicate :strict_model_for, true
```

Comment:
Why not set the default to false here instead of doing so in `#initialize`? This is very minor, but it could lead to some confusion when reading the documentation

Reply:
The boolean argument to `attr_predicate` says whether the predicate is readonly (false) or readwrite (true). It`s not a default value.

Reply:
Ah I see, did not know about that